doc/cascadia/SettingsSchema.md

```diff
@@ -138,6 +138,11 @@ Bindings listed below are per the implementation in `src/cascadia/TerminalApp/Ap
 - moveFocusDown
 - toggleFullscreen
 
+## Example Keys
+- ctrl+1
+- ctrl+pgdn
+- ctrl+pgup
```

Comment:
While we're here, should we just add the following:
```suggestion
- ctrl+alt+shift+pgup
``` to clue people in that <kbd>alt</kbd> and <kbd>shift</kbd> also work?